common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java

```diff
@@ -138,26 +138,14 @@ public static boolean setLogger(@NonNull String identifier,
     /**
      * Get only the required metadata from the DiagnosticContext
      * to plug it in the log lines.
-     * Here we are considering the correlation_id and the thread_name.
+     * Here we are considering the correlation_id and the thread_name. Calls private getDiagnosticContextMetadata with
+     * correlationId from existing RequestContext in DiagnosticContext
      * The need for this is because DiagnosticContext contains additional metadata which is not always required to be logged.
      *
      * @return String The concatenation of thread_name and correlation_id to serve as the required metadata in the log lines.
      */
     public static synchronized String getDiagnosticContextMetadata() {
-        String threadName = DiagnosticContext.INSTANCE.getRequestContext().get(DiagnosticContext.THREAD_NAME);
-        String correlationId = DiagnosticContext.INSTANCE.getRequestContext().get(DiagnosticContext.CORRELATION_ID);
-
-        if (StringUtil.isNullOrEmpty(threadName)) {
-            threadName = UNSET;
-        }
-        if (StringUtil.isNullOrEmpty(correlationId)) {
-            correlationId = UNSET;
-        }
-
-        return DiagnosticContext.THREAD_NAME + " : "
-                + threadName + ", "
-                + DiagnosticContext.CORRELATION_ID + " : "
-                + correlationId;
+        return getDiagnosticContextMetadata(DiagnosticContext.INSTANCE.getRequestContext().get(DiagnosticContext.CORRELATION_ID));
```

Comment:
Seems like now this method only returns correlation id? If that's the case then we should probably update javadoc as well as the method name.

Reply:
It still returns correlationId and thread_name like it used to. As result of refactoring, it now calls into private method passing correlationId from request context.